lte/gateway/deploy/roles/magma/files/update_mme_config_for_sanity.sh

```diff
@@ -87,6 +87,26 @@ function configure_restricted_plmn {
     "$mme_config_file"
 }
 
+function configure_blocked_imei {
+  # Remove default blocked imei(s) from MME configuration file
+  sed -i -e '/BLOCKED_IMEI_LIST/{n;d}' \
+    "$mme_config_file"
+
+  # Configure blocked imei(s) in MME configuration file
+  blocked_imei_config=(
+    '{ TAC="99000482"; SNR="351037" }'
+  )
```

Comment:
can you add a wildcarded SNR case and add a new test (or expand the existing test cases) for that?